src/refactor_nrepl/ns/pprint.clj

```diff
@@ -35,11 +35,19 @@
                             (printf "%s " libspec))))))
                   ordered-libspecs))))
 
-(defn insert-clause-delimiter []
+(defn- insert-clause-delimiter []
   (if (:insert-newline-after-require *config*)
     (println)
     (print " ")))
 
+(defn- pprint-libspec [libspec]
+  ;; If a vector gets too long `pprint` will print one element per line.
+  ;; This puts `:as` and `:refer` on their own line, which causes the ns form
+  ;; to take up too much vertical space.
+  (printf (str/replace (with-out-str (pprint libspec))
+                       (re-pattern (str "(:as|refer)" (System/lineSeparator)))
```

Comment:
Looks like `refer` (no colon) would be matched, might fail in some corner case

Reply:
Fixed, thanks!